examples/conll02_rnn.py

```diff
@@ -0,0 +1,282 @@
+    testb_x, testb_y = to_ngrams(testb_sentences, testb_labels, ngram)
+
+    # generate vocabulary for x and y
+    vocab = to_vocabulary(train_sentences) | to_vocabulary(testa_sentences) | to_vocabulary(testb_sentences)
+    vocab_label = to_vocabulary(train_labels)
+
+    # generate a dictionary mapped from word / label to its index
+    vocab_index_dict = {w: i for i, w in enumerate(vocab)}
+    label_index_dict = {l: i for i, l in enumerate(vocab_label)}
+
+    # convert from words to words' indexes
+    train_x = to_vocab_index(vocab_index_dict, train_x)
+    testa_x = to_vocab_index(vocab_index_dict, testa_x)
+    testb_x = to_vocab_index(vocab_index_dict, testb_x)
+
+    # convert from labels to labels' indexes
+    train_y = to_label_index(label_index_dict, train_y)
+    testa_y = to_label_index(label_index_dict, testa_y)
+    testb_y = to_label_index(label_index_dict, testb_y)
+
+    bgn_label_idx = label_index_dict['__BGN__']
+
+    return train_x, train_y, testa_x, testb_x, testa_y, testb_y, vocab, vocab_label, bgn_label_idx
+
+
+rnn = recurrent.GRU
+embedding_size = 256
+hidden_size = 256
+batch_size = 128
+epochs = 20
+ngram = 3
+
+# download CoNLL 2002 dataset from official website and unzip if local file doesn't exist
+get_file('http://www.cnts.ua.ac.be/conll2002/ner.tgz')
+
+train_x, train_y, testa_x, testb_x, testa_y, testb_y, vocab, vocab_label, bgn_label_idx = get_data('./ner/data', ngram)
+# Reserve 0 for masking via pad_sequences
+vocab_size = len(vocab) + 1
+vocab_label_size = len(vocab_label)
+print('vocab_size = ', vocab_size)
+print('vocab_label_size =', vocab_label_size)
+
+# pad x
+print('Padding train data...')
+train_x_pad = sequence.pad_sequences(train_x, maxlen=ngram)
+print('Padding test data a...')
+testa_x_pad = sequence.pad_sequences(testa_x, maxlen=ngram)
+print('Padding test data b...')
+testb_x_pad = sequence.pad_sequences(testb_x, maxlen=ngram)
+
+print('train_x.shape = {}'.format(train_x_pad.shape))
+print('testa_x.shape = {}'.format(testa_x_pad.shape))
+print('testb_x.shape = {}'.format(testb_x_pad.shape))
+
+# convert class vectors to binary class matrices
+train_y_categorical = np_utils.to_categorical(train_y, vocab_label_size)
+testa_y_categorical = np_utils.to_categorical(testa_y, vocab_label_size)
+testb_y_categorical = np_utils.to_categorical(testb_y, vocab_label_size)
+
+print('train_y_categorical.shape = {}'.format(train_y_categorical.shape))
+print('testa_y_categorical.shape = {}'.format(testa_y_categorical.shape))
+print('testb_y_categorical.shape = {}'.format(testb_y_categorical.shape))
+
+# build model
+print('Build model...')
+model = Sequential()
+model.add(Embedding(vocab_size, embedding_size, mask_zero=True))
+model.add(rnn(hidden_size, return_sequences=False))
+model.add(Dropout(0.5))
+model.add(Dense(vocab_label_size))
+model.add(Activation('softmax'))
+model.compile(optimizer='adam', loss='categorical_crossentropy', class_mode='categorical')
+
```

Comment:
@xingdi-eric-yuan shouldn't Dense have two init parameters? Is this a bug in your code, or has this changed in the latest keras master?